fe1-web/store/reducers/SocialReducer.ts

```diff
@@ -53,7 +53,7 @@ const socialSlice = createSlice({
           };
         }
 
-        state.byLaoId[laoId].allChirps.push(chirp);
+        state.byLaoId[laoId].allChirps.unshift(chirp);
```

Comment:
no need to address it in this PR, but just be aware that this will be buggy as you could receive multiple times the same chirp.

Reply:
Is it the fact that I use unshift the problem or is it about the fact that this is a list ?
Anyway, this list is only temporary and will be replaced by something more suitable.